test/apps/gups.c

```diff
@@ -195,64 +195,64 @@ int SHMEMRandomAccess(void);
 
```

Comment:
It looks like gups is 2 spaces whereas SOS is 4.  Doesn't matter to me, just checking whether that is intentional.